app/models/user.rb

```diff
@@ -17,4 +19,8 @@ class User < ApplicationRecord
   def convert_email_to_downcase
     self.email = email.downcase
```

Comment:
I saw this in someones PR and now it's merged. Why do you do that `email.downcase`??